.github/workflows/build.yaml

```diff
@@ -35,38 +35,95 @@ jobs:
         with:
           node-version: "16"
 
+      - name: Fetch dependencies from cache
+        id: cache-node-modules
+        uses: actions/cache@v3
+        with:
+          path: "**/node_modules"
+          key: yarn-build-${{ hashFiles('**yarn.lock') }}
+          restore-keys: |
+            yarn-build-
+
+      - name: Install dependencies
+        if: steps.cache-node-modules.outputs.cache-hit != 'true'
+        run: SKIP_SUBMODULE_DEPS=1 yarn --frozen-lockfile
+
+      - name: Format files with Prettier
+        run: yarn fmt
+
+  lint-helm:
+    name: Lint Helm chart
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v23.2
+        with:
+          files: |
+            ci/helm-chart/**
+
       - name: Install helm
+        if: steps.changed-files.outputs.any_changed == 'true'
         uses: azure/setup-helm@v3.3
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Install helm kubeval plugin
+        if: steps.changed-files.outputs.any_changed == 'true'
         run: helm plugin install https://github.com/instrumenta/helm-kubeval
 
+      - name: Lint Helm chart
+        if: steps.changed-files.outputs.any_changed == 'true'
+        run: helm kubeval ci/helm-chart
+
+  lint-ts:
+    name: Lint TypeScript files
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v23.2
+        with:
+          files: |
+            **/*.ts
+            **/*.js
+          files_ignore: |
+            lib/vscode/**
+
+      - name: Install Node.js v16
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16"
+
       - name: Fetch dependencies from cache
+        if: steps.changed-files.outputs.any_changed == 'true'
         id: cache-node-modules
         uses: actions/cache@v3
         with:
           path: "**/node_modules"
-          key: yarn-build-${{ hashFiles('**/yarn.lock') }}
+          key: yarn-build-${{ hashFiles('**yarn.lock') }}
```

Comment:
Was the / removal here intentional? (Same for the other keys.)

Reply:
Bad search and replace 🤦🏼‍♂️

Reply:
Fixed